tests/validators/tests.py

```diff
@@ -205,6 +205,11 @@
 
     (URLValidator(EXTENDED_SCHEMES), 'file://localhost/path', None),
     (URLValidator(EXTENDED_SCHEMES), 'git://example.com/', None),
+    (URLValidator(EXTENDED_SCHEMES), 'ssh://example.com/', None),
+    (URLValidator(EXTENDED_SCHEMES), 'git+ssh://git@github.com/example/hg-git.git', None),
+
+    # Empty schemes should not be accepted
+    (URLValidator(schemes=['']), '://www.example.com', ValidationError),
```

Comment:
I guess what I'm saying is that providing `schemes=['']` should be an error itself, in which case the regular expression change isn't really needed since it's not used to validate schemes as noted by the comment "# scheme is validated separately". Can we just add the plus sign to the allowed characters and move on to more important things?